packages/vuetify/src/components/VList/VListItem.tsx

```diff
@@ -153,6 +157,12 @@ export const VListItem = genericComponent<VListItemSlots>()({
       props.link !== false &&
       (props.link || link.isClickable.value || isSelectable.value)
     )
+    const isKeyboardFocused = computed(() =>
+      list &&
+      list.navigationStrategy.value === 'track' &&
+      props.index !== undefined &&
+      list.keyboardFocusedIndex.value === props.index
+    )
```

Comment:
`keyboard*` might suggest a general use case, but could also introduce friction when troubleshooting.

What do you think about those?
- `isTracked`
- `list.trackingIndex`